bin/3scale_backend_worker

```diff
@@ -21,6 +21,10 @@ if !File.writable? options[:dir]
   FileUtils.mkdir_p options[:dir]
 end
 
-Daemons.run_proc('3scale_backend_worker', options) do
+if ARGV.delete '--debug'
   ThreeScale::Backend::Worker.work
```

Comment:
Here, better use:

`ThreeScale::Backend::Worker.work(job_fetcher: ThreeScale::Backend::JobFetcher.new(fetch_timeout: 1))`

You would need to `require '3scale/backend/job_fetcher'`first.

This way the worker actually stops when we click the stop button.

Reply:
@jlledom What would happen if the `fetch_timeout` is not specified? Would it wait forever and hence the process would never stop? Or is it that this timeout is just too big, so eventually it would stop.

I think we need some comment on this to make it clear why this is needed.